files/usr/etc/systemd/system/rpm-ostreed-automatic.timer.conf.d/override.conf

```diff
@@ -0,0 +1,3 @@
+[Timer]
```

Comment:
this file needs to be in `rpm-ostreed-automatic.timer.d` not in `rpm-ostreed-automatic.timer.conf.d`

Reply:
thank you!